### DISCUSSION TITLE
Can I add tsParticles to an existing app?

### DISCUSSION THREAD

**Discussion by rupert-denton** [matteobruni/tsparticles#5239] [Q&A]
I am building a small ed-tech app and would love to use tsParticles to make it more engaging. I need it mainly for when a correct button is clicked and when a level increases. 

I am building with vite-react-typescript. Thanks!

**Comment by matteobruni** [comment]
Here is the React component readme, follow the instruction and you should be ready.

https://github.com/tsparticles/react#readme
